data/pages.yml

```diff
@@ -232,3 +232,9 @@
       url: "adding-new-features"
     - title: "Repositories"
       url: "repositories"
+
+- title: "Glossary"
```

Comment:
- title: "Glossary"
  url: "glossary"
  pages:
    - title: "This Is The Actual Glossary"
      url: "oh-hi-im-the-glossary"